src/Coriolis/constant_coriolis.jl

```diff
@@ -0,0 +1,82 @@
+using Oceananigans.Grids: ZDirection, validate_unit_vector
+
+"""
+    ConstantCoriolis{FT} <: AbstractRotation
+
+A Coriolis implementation that accounts for the locally vertical and possibly both local horizontal
+components of a constant rotation vector. A more general implementation of [`FPlane`](@ref), which only
+accounts for the locally vertical component.
+"""
+struct ConstantCoriolis{FT} <: AbstractRotation
+    fx :: FT
+    fy :: FT
+    fz :: FT
+end
+
+"""
+    ConstantCoriolis([FT=Float64;] fx=nothing, fy=nothing, fz=nothing,
+                                f=nothing, rotation_axis=ZDirection(), 
+                                rotation_rate=Ω_Earth, latitude=nothing)
+
+Returns a parameter object for a constant rotation decomposed into the `x`, `y` and `z` directions.
+In oceanography the components `x`, `y`, `z` correspond to the directions east, north, and up. This
+rotation can be specified in three different ways:
+
+- Specifying all components `fx`, `fy` and `fz` directly.
+- Specifying the Coriolis frequency `f` and (optionally) a `rotation_axis` (which defaults to the
```

Comment:
Might be less controversial to call it the Coriolis "parameter" --- most people interpret `f` as a vorticity (eg the planetary vorticity, or the vorticity associated with solid body rotation); even though it also sets the time-scale for inertial waves that are perturbations to solid body rotation.

Reply:
Done